web/styles/zulip.css

```diff
@@ -1720,9 +1720,19 @@ td.pointer {
 }
 
 .sender_name {
-    display: inline-block;
+    display: inline-flex;
     font-weight: 700;
     color: var(--color-text-sender-name);
+
+    .sender_name_text {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+    }
+
+    .status-emoji {
+        margin-left: 6px;
```

Comment:
Since you're displaying this as `inline-flex`, maybe use the `column-gap` property on the flex definition instead?

Reply:
(Not sure why I didn't do that myself, as I think this comes from code I introduced with #24994.)

Reply:
Yeah, done.